lib/bemhtml/runtime/index.js

```diff
@@ -389,6 +390,36 @@ BEMHTML.prototype.runOne = function runOne(json) {
   return res;
 };
 
+BEMHTML.prototype.tryRun = function tryRun(context, ent) {
+  var res;
+
+  try {
+    res = ent.run(context, ent);
+  } catch (e) {
+    console.error(e);
+
+    if (this.options.debug !== true)
+      return '';
+
+    res = this.buildRenderError(context, e);
+  }
+
+  return res;
+};
+
+BEMHTML.prototype.buildRenderError = function buildRenderError(context, e) {
```

Comment:
Why we decide to left this in production mode? Is it possible to drop this function in production?